src/tasks/mixin/navigation_mixin.py

```
@@ -100,96 +129,79 @@ def check_target():
+        self.send_key_down("w")  # 确认使用send_key：w为方向移动键，不属于游戏可配置热键，用于持续移动
 
         try:
-
             while True:
-
-                # 到达目标
                 reached = check_target()
 
                 if reached:
-                    self.send_key_up("w")
+                    self.send_key_up("w")  # 确认使用send_key：释放方向键
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)